lib/soapy_cake/admin.rb

```diff
@@ -102,6 +102,10 @@ def affiliate_signup(opts = {})
       run Request.new(:admin, :signup, :affiliate, opts)
     end
 
+    def media_types(*)
```

Comment:
Do we actually use the variable-length argument array anywhere?  If not, it seems it a little weird to me that we accept arguments and then just throw them away.  Just curious if there's a reason for that.